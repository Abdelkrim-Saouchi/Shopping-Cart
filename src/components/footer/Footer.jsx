import styles from './Footer.module.css';

const Footer = () => {
  return (
    <footer className={'container ' + styles.footer}>
      Copyright &#169;
      <a href="https://github.com/Abdelkrim-Saouchi"> Krimothiazine</a>
    </footer>
  );
};

export default Footer;
