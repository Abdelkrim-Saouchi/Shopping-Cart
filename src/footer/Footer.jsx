const Footer = () => {
  return (
    <footer className="container">
      Copyright &#169;
      <a href="https://github.com/karim-saou"> Krimothiazine</a>
    </footer>
  );
};

export default Footer;
