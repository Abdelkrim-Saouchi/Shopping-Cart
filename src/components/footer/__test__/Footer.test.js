import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';

import { BrowserRouter } from 'react-router-dom';
import Footer from '../Footer';

test('renders homepage correctly', () => {
  const { container } = render(<Footer />, { wrapper: BrowserRouter });
  expect(container).toMatchSnapshot();
});
