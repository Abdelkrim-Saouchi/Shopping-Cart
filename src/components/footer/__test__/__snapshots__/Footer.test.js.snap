// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage correctly 1`] = `
<div>
  <footer
    class="container footer"
  >
    Copyright Â©
    <a
      href="https://github.com/Abdelkrim-Saouchi"
    >
       Krimothiazine
    </a>
  </footer>
</div>
`;
